Dockerfile

```diff
@@ -1,13 +1,13 @@
 # First stage obtains the list of certificates.
-FROM --platform=$BUILDPLATFORM debian:stable-slim@sha256:64bc71feaa7ec2ac758a6a3a37c0f0d6ebccf0a45e3f5af1f1d3b5d4cb316b29 AS build
-RUN apt-get update && apt-get -y install ca-certificates
+FROM --platform=$BUILDPLATFORM alpine:3.20 AS build
+RUN apk --no-cache add ca-certificates
```

Comment:
I'd suggest to get rid of this stage and the subsequent `COPY --from=build /etc/ssl/certs/ca-certificates.crt ...` and just install ca-certificates on the final layer. I _think_ `apk --no-cache` should leave little leftovers, if at all, and will lower the overall complexity.

Reply:
I think it might be worth leaving the `build` target in for now.

The build for linux/arm64/v8 [fails](https://drone.grafana.net/grafana/synthetic-monitoring-agent/2291/1/8) to run `apk --no-cache add ca-certificates` when building the release.

Reply:
That sounds a bit unsettling, but might be due to buildx shenanigans. I think we can revisit that when we fully migrate the build container pipeline to GHA.

Reply:
From mem's [suggestion](https://github.com/grafana/synthetic-monitoring-agent/pull/875#discussion_r1759574118), it looks like it works if we use `ca-certificates-bundle` instead